File: frontend/src/domain/portalPublicationStatus.ts
export enum PortalPublicationStatus {
  PUBLISHED = "PUBLISHED",
  UNPUBLISHED = "UNPUBLISHED",
  WITHDRAWN = "WITHDRAWN",
}

export const PortalPublicationStatusLabel: Record<
  PortalPublicationStatus,
  string
> = {
  [PortalPublicationStatus.PUBLISHED]: "Veröffentlicht",
  [PortalPublicationStatus.UNPUBLISHED]: "Unveröffentlicht",
  [PortalPublicationStatus.WITHDRAWN]: "Zurückgezogen",
}

File: backend/src/main/java/de/bund/digitalservice/ris/caselaw/domain/PortalPublicationStatus.java
package de.bund.digitalservice.ris.caselaw.domain;

public enum PortalPublicationStatus {
  UNPUBLISHED("Unveröffentlicht"),
  PUBLISHED("Veröffentlicht"),
  WITHDRAWN("Zurückgezogen");

  public final String humanReadable;

  PortalPublicationStatus(String humanReadable) {
    this.humanReadable = humanReadable;
  }
}
